components/src/lib/hook/useModalHandle.ts

```diff
@@ -0,0 +1,16 @@
+import { useState } from "react";
+function useModalHandler() {
+  const [modalOpen, setModalOpen] = useState(false);
+
+  const openModal = () => {
+    setModalOpen(true);
+  };
+
+  const closeModal = () => {
+    setModalOpen(false);
+  };
+
+  return { modalOpen, openModal, closeModal };
```

Comment:
modalOpen과 openModal의 차이점은 무엇일까요?!

보통 boolean을 반환하는 상태에 대해서는 is, has 접두사를 많이 사용하더라구요!
isModalOpened 혹은 isOpened와 같은 식으로 사용하면 어떨까 싶어요 :)
정 is/has가 별로라면 modalOpened 정도 사용해도 괜찮을 것 같아요~!

```suggestion
  return { isModalOpened, openModal, closeModal };
```